joomla/joomla-cms
# [newsfeed modal field] Adds Create News Feed button and other improvements

Pull Request for Improvement.
### Summary of Changes

This PR is the sequence of https://github.com/joomla/joomla-cms/pull/11830 but for now for newsfeed modal.

For list of changes see https://github.com/joomla/joomla-cms/pull/11830 (same changes, but for newsfeed now).

Similiar PR: 
- Article: https://github.com/joomla/joomla-cms/pull/11830
- Category: https://github.com/joomla/joomla-cms/pull/11857
- Contact: https://github.com/joomla/joomla-cms/pull/11858
- Newsfeed: https://github.com/joomla/joomla-cms/pull/11862 (this one)
### Animated Gif (click to view in full screen)

![modal-newsfeed](https://cloud.githubusercontent.com/assets/9630530/18131118/907dab24-6f89-11e6-97e0-84da0df6e14e.gif)
### Testing Instructions
- Use latest staging
- Apply patch https://github.com/joomla/joomla-cms/pull/11830 and then this patch
- Create a Menu item Single News Feed  type
- Test all the "Edit", "Create", "Select" and "Clear" buttons  (as animated gif)
- In a multilanguage install edit a news feed and try the modal button in the associations tab
- Test all the "Edit", "Create", "Select" and "Clear" buttons.
- Code review to check all is ok.
### Documentation Changes Required

None.
